test/Npgsql.Tests/Types/RangeTests.cs

```diff
@@ -247,13 +247,32 @@ await using (var conn = await OpenConnectionAsync())
             isNpgsqlDbTypeInferredFromClrType: false);
     }
 
+    NpgsqlDataSource DataSourceWithRanges { get; set; } = default!;
+
     [OneTimeSetUp]
     public async Task OneTimeSetUp()
     {
+        DataSourceWithRanges = CreateDataSourceWithRanges();
         using var conn = await OpenConnectionAsync();
         MinimumPgVersion(conn, "9.2.0");
     }
 
+    [OneTimeTearDown]
+    public async Task TearDown()
+    {
+        if (DataSourceWithRanges is not null)
```

Comment:
When is this expected to be null?

Reply:
Maybe if we fail while creating `DataSource` in `OneTimeSetUp`, though I'm not sure if NUnit is even going to call `TearDown` in that case. But anyway, that's just to make sure we don't fail here in any case.

Reply:
I think if there's an exception from OneTimeSetUp, there's no TearDown - should be able to simplify.

(note that all the tests in this class assume it's non-null without checking, it's a bit weird for only TearDown to do so)

Reply:
Removed null check